pkg/backend/openshiftcluster.go

```diff
@@ -76,7 +77,21 @@ func (ocb *openShiftClusterBackend) handle(ctx context.Context, log *logrus.Entr
 	stop := ocb.heartbeat(ctx, cancel, log, doc)
 	defer stop()
 
-	m, err := openshiftcluster.NewManager(log, ocb.env, ocb.db.OpenShiftClusters, ocb.billing, doc)
+	r, err := azure.ParseResourceID(doc.OpenShiftCluster.ID)
+	if err != nil {
+		return err
+	}
+
+	subscriptionDoc, err := ocb.db.Subscriptions.Get(ctx, r.SubscriptionID)
+	if err != nil {
+		return err
+	}
+
+	if subscriptionDoc == nil {
+		return fmt.Errorf("Subscription document for '%s' not found", r.SubscriptionID)
+	}
+
+	m, err := openshiftcluster.NewManager(log, ocb.env, ocb.db.OpenShiftClusters, ocb.billing, doc, subscriptionDoc)
```

Comment:
@jim-minter I've made this section match that in @olga-mir's [PR](https://github.com/Azure/ARO-RP/pull/871/files#diff-b511bf730e3452407da6057dd9cfc99cR80-R94)

Reply:
Looks like that PR was merged against this file so is this diff out of date? Can you rebase it against master?

Reply:
Rebased